templates/user/dashboard/feeds.tmpl

```diff
@@ -24,7 +24,8 @@
 						{{else if eq .GetOpType 8}}
 							{{$.i18n.Tr "action.transfer_repo" .GetContent .GetRepoLink .ShortRepoPath | Str2html}}
 						{{else if eq .GetOpType 9}}
-							{{$.i18n.Tr "action.push_tag" .GetRepoLink .GetBranch .ShortRepoPath | Str2html}}
+							{{ $branchLink := .GetBranch | EscapePound | Escape}}
```

Comment:
Shouldn't this be also the non-legacy path like above https://github.com/go-gitea/gitea/pull/3767/files#diff-06961515e08a5e52a731517765266c80R16

Reply:
Comment from above also applies here.